test/r2.md

```diff
@@ -439,3 +441,27 @@ The following command with broken `single_cortx.iso` should be failed
 ```bash
 provisioner set_swupgrade_repo 2.0.1 --source=/opt/iso/single_cortx.iso
 ```
+
+
+## EOS-17828: Port adjustments post discussions for 3 node
+
+Step 1: Ensure firewall component is present in all the nodes of a 3 node VM setup. If not, install with the below command
+```bash
+salt '*' state.apply components.system.firewall
+```
+
+Step 2: Verify the services with corresponding ports, are mapped to all the zones with the below command.
+```bash
+firewall-cmd --list-all
+```
+
+Step 3: Verify if the services are mapped to the zones on all the nodes.
+```bash
+ssh srvnode-2 firewall-cmd --list-all
```

Comment:
Similar to above:
```
for node in srvnode-1 srvnode-2 srvndoe-3
do
    for zone in "management-zone" "public-data-zone" 
    do
        ssh ${node} firewall-cmd --list-all --zone ${zone}
    done
done
```

After this a separate `Step 2` shall no more be necessary.

Reply:
Comment addressed.